### PR TITLE
Fix/rss ecomm 3 23 fix address styles

### PR DESCRIPTION
**Pull request by Maksim99745**
fix/RSS-ECOMM-3_23-fix-address-styles

#### 🤔 This is a ...

- [x] 🌟 New task

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [x] 📝 I have commented my code, particularly in hard-to-understand areas.
- [x] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.
- [x] 👌 Have at least 1 approve.
- [x] 🌳 Branch from `develop`/`release/...` and target to `develop`/`release/...`.

### CODE REVIEW

File: src/core/api/api.service.ts [hunk 1]
```diff
@@ -32,7 +32,6 @@ export class ApiService {
 
     if (token) {
       userLoadingSignal.value = true;
-
```
Comment (Gromov-Roman) [inline src/core/api/api.service.ts]: return empty line pls

File: toMaksim-3.patch
Comment (Gromov-Roman) [inline toMaksim-3.patch:1]: Why is this file here?

Comment (Gromov-Roman) [reply toMaksim-3.patch:1]: I believe that you can remove it

**Review by JsPowWow — APPROVED** [review]
